app/src/main/java/com/github/sdp_begreen/begreen/models/PhotoMetadata.kt

```diff
@@ -4,41 +4,19 @@ import android.os.Parcel
 import android.os.Parcelable
 
 
-data class PhotoMetadata(var pictureId: String? = null, val title: String? = null,
-                         val takenOn: ParcelableDate? = null, val takenBy: String? = null,
-                         val category: String? = null, val description: String? = null)
-    : Parcelable {
-    constructor(parcel: Parcel) : this(
-        parcel.readString(),
-        parcel.readString(),
-        parcel.readParcelable(ParcelableDate::class.java.classLoader),
-        parcel.readString(),
-        parcel.readString(),
-        parcel.readString()
-    )
+abstract class PhotoMetadata(
```

Comment:
Is there any point in keeping the abstract class and not deleting it completely? It seems to pollute the code more than anything else. I'm not a big fan of abstract classes (unless there is a very good reason of course).